CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # CHANGELOG
 
+## 3.4.2
```

Comment:
We should handle adding feature and version upgrade separately. Can you remove this version upgrade to 3.4.2 from your PR?